EngineLayer/ModernSearch/ModernSearchEngine.cs

```diff
@@ -36,20 +33,17 @@ public class ModernSearchEngine : MetaMorpheusEngine
 
         #region Public Constructors
 
-        public ModernSearchEngine(Psm[][] globalPsms, Ms2ScanWithSpecificMass[] listOfSortedms2Scans, List<CompactPeptide> peptideIndex, float[] keys, List<int>[] fragmentIndex, Tolerance fragmentTolerance, List<MassDiffAcceptor> searchModes, List<string> nestedIds, bool addCompIons, List<ProductType> lp, double scoreCutoff, int currentPartition, int totalPartitions) : base(nestedIds)
+        public ModernSearchEngine(Psm[][] globalPsms, Ms2ScanWithSpecificMass[] listOfSortedms2Scans, List<CompactPeptide> peptideIndex, float[] keys, List<int>[] fragmentIndex, List<ProductType> lp, int currentPartition, CommonParameters commonParameters, SearchParameters searchParameters, List<string> nestedIds) : base(nestedIds)
```

Comment:
instead of passing SearchParameters , just pass MassDiffAcceptors and AddCompIons, like in the classic search engine